css/css-ui/appearance-cssom-001.html

```diff
@@ -2,8 +2,11 @@
 <title>CSS Basic User Interface Test: appearance CSSOM</title>
 <link rel="help" href="https://drafts.csswg.org/css-ui-4/#appearance-switching">
 <meta name="assert" content="CSSOM for the appearance/-webkit-appearance property is correct.">
+<meta name="variant" content="">
+<meta name="variant" content="?exclude=Invalid">
```

Comment:
@nt1m this means there's some overlap between the two; it might be better to have `?include=Invalid` instead of `""`.

Reply:
https://github.com/web-platform-tests/wpt/pull/39317